<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>729</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe MDL2 Assets</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LIBRARY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MY ACCOUNT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel reservation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter book name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert an ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert an ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>521</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>119</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>119</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Reserve</string>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>listWidget_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionlog_out"/>
    <addaction name="actiondelete_account"/>
    <addaction name="actionchange_password"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlog_out">
   <property name="text">
    <string>log out</string>
   </property>
  </action>
  <action name="actiondelete_account">
   <property name="text">
    <string>delete account</string>
   </property>
  </action>
  <action name="actionsearch_for_books">
   <property name="text">
    <string>search for books</string>
   </property>
  </action>
  <action name="actioncancel_reservation">
   <property name="text">
    <string>cancel reservation</string>
   </property>
  </action>
  <action name="actionchange_password">
   <property name="text">
    <string>change password</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
